README.markdown

```diff
@@ -8,17 +8,38 @@ Deviating from the guide is allowed, but be prepared to defend your decision in
 In general we prefer clarity over cleverness.
 Try to write code that is easy to understand, debug, and modify.
 
+> [Haskell 98](https://www.haskell.org/onlinereport/) ought to be enough for anybody.
+
 ## Table of contents
 
-- [Formatting](#formatting)
+- [Avoid compiler warnings](#avoid-compiler-warnings)
+- [Apply HLint suggestions](#apply-hlint-suggestions)
+- [Format with Hindent](#format-with-hindent)
 - [Prefer `let` over `where`](#prefer-let-over-where)
 - [Avoid pure `do`](#avoid-pure-do)
 - [Avoid writing partial functions](#avoid-writing-partial-functions)
 - [Avoid using partial functions](#avoid-using-partial-functions)
 - [Avoid throwing exceptions](#avoid-throwing-exceptions)
 - [Prefer qualified imports](#prefer-qualified-imports)
 
-## Formatting
+## Avoid compiler warnings
+
+Code should compile with `-Weverything` and `-Werror`.
+We ignore a few warnings:
+
+- `-Wno-implicit-prelude`
+- `-Wno-safe`
+- `-Wno-unsafe`
+
+## Apply HLint suggestions
```

Comment:
This reminds me, should we have a statement about checking hlint and hindent files into version control? I'd prefer to avoid anything specific about git. If you want to use [darcs](http://darcs.net/), or [hg](https://www.mercurial-scm.org/) then go right ahead..

Reply:
Great question! I was actually just reviewing my [Haskell package checklist](https://github.com/tfausak/haskell-package-checklist). I was trying to determine how many of those suggestions are applicable here. Do we want to suggest more "project management" type stuff?

Reply:
👍 I agree. I'm working under the assumption that most of these suggestions are meant to be applied to an existing codebase/package. So suggesting Git (or Stack or hpack or …) is ultimately irrelevant.